### PR TITLE
Translate accountants page

### LINKED ISSUE #1583: I18n Spanish: Accounting
## What are we trying to do?

This ticket is part of the DARIA translation effort. The goal is to implement all our text in both English and Spanish through the i18n* process. 

This ticket is for the Accounting views.  
Code link:  
app/views/accountants/index.html.erb  
app/views/accountants/_search_form.html.erb  
app/views/accountants/_table_content.html.erb  

The current English text should be moved to `config/locales/en.yml`,   
and new Spanish text should be added to `config/locales/es.yml`.

**Google translate is perfectly acceptable!**  
PRs will be reviewed by a fluent CM to correct any weirdness.

### Steps

For each view  
 - find each instance of displayed text on the view
   `<%= f.date_field :appointment_date, label: 'Appointment date', autocomplete: 'off', help: "Approx gestation at appt: #{patient.last_menstrual_period_at_appt}" %>`
 - replace it with a call to i18n `t` function with a unique identifier based on the file path & field
   `<%= f.date_field :appointment_date, label: t('patient.dashboard.appt_date'), autocomplete: 'off', help: t('patient.dashboard.approx_gestation', gest: patient.last_menstrual_period_at_appt) %>`
 - add a config entry for each language with that identifier as the key, and the language line as the value.
   `en.yml`:
   ```
   patient:
     dashboard:
       approx_gestation: 'Approx gestation at appt: %{gest}'
       appt_date: Appointment date
    ```  
    `es.yml`:
    ```
    patient:
      dashboard:
        approx_gestation: 'Aprox la gestación en la cita: %{gest}'
        appt_date: Día de la cita
    ```

### Completed View Example

For an example of a completed view translation, see [the footer view i18n PR](https://github.com/DCAFEngineering/dcaf_case_management/pull/1547/files).

### Formatting in the YAML files

Each view should have a it's own section. If you see the same key being used across Some commonly used values can go under a shared secton. `app/views/patients/_patient_dashboard.html.erb` keys go under `patient:` -> `dashboard:`.

### Finalizing with i18n-tasks

After you finish all the keys, you should run the i18n checker before committing! This sorts the keys in a standard way and checks for missing keys.

`i18n-tasks normalize`
`i18n-tasks health` - address any issues this highlights!

[More info on using i18n-tasks](https://github.com/glebm/i18n-tasks)

\* 'i18n' is shorthand for 'Internationalization'. [More info on i18n in Rails](https://guides.rubyonrails.org/i18n.html)

Part of #1382

### PR DESCRIPTION
**Pull request by colinxfleming**
I rule and have completed some work on Case Manager that's ready for review!

Run the accountants workflow thru i18n.

This pull request makes the following changes:
* i18n-icize the accountants page

image

It relates to the following issue #s: 
* Fixes #1583

### CODE REVIEW

File: app/views/accountants/_table_content.html.erb [hunk 1]
```diff
@@ -1,22 +1,22 @@
...
-        <th></th>
+  <thead>
+    <tr>
+      <th><%= t 'accountants.table_content.clinic' %></th>
+      <th><%= t 'accountants.table_content.pt_initials' %></th>
+      <th><%= t 'accountants.table_content.fund_payout', fund: FUND %></th>
+      <th><%= t 'accountants.table_content.weeks_at_procedure' %></th>
+      <th><%= t 'accountants.table_content.pledge_sent' %></th>
+      <th><%= t 'accountants.table_content.check_number' %></th>
+      <th><%= t 'accountants.table_content.date_of_check' %></th>
```
Comment (lomky) [inline app/views/accountants/_table_content.html.erb]: Suggest moving most of these to a shared - I just ran into most of them in the patient pledge PR! #1621 

If they don't feel like a good case for a 'shared' namespace, I think it's more appropriate to name them for the pledge tree, but I don't feel strongly.

File: config/locales/es.yml [hunk 2]
```diff
@@ -280,3 +280,16 @@ es:
     resolved_without_fund: Esto se utiliza para indicar que un paciente ya no necesita o quiere nuestros servicios.
     status_definition: Definición de estado
     urgent_cases: Estos son pacientes que requieren un poco más de atención. Esta lista es compartido entre todos los administradores de casos que trabajan en una sola línea. Los pacientes son eliminados de esta lista automáticamente después de que estén marcados con “compromiso enviado”, o marcado como “resuelto sin ayuda”.
+  accountants:
+    title: Tablero contable
```
Comment (montanezp) [inline config/locales/es.yml]: Tablero de contador

File: app/views/accountants/_table_content.html.erb (hunk 1 above)
Comment (colinxfleming) [reply app/views/accountants/_table_content.html.erb]: These might be specialized to accountants, but if they're not I'll move them to the mongoid attributes namespace. Great call.

Comment (colinxfleming) [reply app/views/accountants/_table_content.html.erb]: After taking a stab at this the language is like EVER so slightly different, in such a way that I don't think this is a free basket rn. My vote is that we go with the redundancy for now in the name of expedience.

Comment (colinxfleming) [reply app/views/accountants/_table_content.html.erb]: (I will happily commit to cleaning this mess up.)

**Pull request merged**